gradle.properties

```diff
@@ -7,4 +7,4 @@ springSecurityVersion=5.3.1.RELEASE
 springDataVersion=2.2.6.RELEASE
 jacksonVersion=2.10.3
 log4jVersion=2.13.1
-elasticVersion=7.1.1
+elasticVersion=7.9.0
```

Comment:
is there a 7.7.x version? the highest on aws right now is 7.7 and the client are forward but not backwards compatible